README.md

```diff
@@ -56,15 +56,15 @@ Thanks to those, all you need to do is run them in succession and you should hav
 
 This file makes temporary folders and files to facilitate the setup. It also makes sure you have `premake5` to generate the project files and that the submodules are initialized and cloned (helpful for those who forgot to clone with `--recurse-submodules`).
 
-On Windows it will ask you for the paths of the **x86** and **x64** versions of Qt and OpenSSL. You don't need both of the versions; you can just specify one and later on build for that specific platform.
+On Windows it will ask you for the paths of the **x86** and **x64** versions of Qt and OpenSSL. You don't need both of the versions; you can just specify one and only build for that specific platform.
 
 > Make sure you use forward slashes instead of the backslashes.
 
 #### `generate_project_files.sh`
 
 As the name suggests, this script generates the necessary files by using [Premake](https://premake.github.io/).
 
-You can provide an action like you do with premake, but the script is configured such that it generates file for
+You can provide an action like you do with premake, but the script is configured such that it (by default) generates files for
```

Comment:
This sentence is a bit clunky. I think "by default" should be right after ", but" without brackets. I.E.
...but by default the script...

Reply:
The script is configured to do something by default; it's not configured by default to do something. But yeah I guess I'll replace it with something shorter and more elegant.